lib/archethic/utils.ex

```diff
@@ -997,4 +993,27 @@ defmodule Archethic.Utils do
     :crypto.hash(:sha256, data_to_digest)
     |> Base.encode16()
   end
+
+  @spec await_confirmation(tx_address :: binary(), list(Node.t())) ::
+          :ok | {:error, :network_issue}
+  def await_confirmation(tx_address, [node | rest]) do
+    case P2P.send_message(node, %GetTransactionSummary{address: tx_address}) do
+      {:ok,
+       %TransactionSummaryMessage{transaction_summary: %TransactionSummary{address: ^tx_address}}} ->
+        :ok
+
+      {:ok, %NotFound{}} ->
+        Process.sleep(200)
+        await_confirmation(tx_address, [node | rest])
```

Comment:
as discussed, infinite loop to remove here

Reply:
After discussion with @tenmoves, we should not remove it right away.  
This is still a required "infinite loop". But we can make it "finite". 
This is because the time for the 1st node to replicate the transaction needs be awaiting. However we should use some backoff method with cap to know if we need to switch node in the list (recursive function).

Reply:
We can use Task.async_nolink with Task. yieldall

Reply:
> No sure because it depends on the time of replication. Even if you use a quorum, we would have to retry after some time. Like on the bootstrap.

Yes, but instead to ask to only one node we request to multiple nodes, so if the first requested node is out of sync it's not a problem.
But of course we will need the retry, but no need to manage the nodes to request

Reply:
we can use Task .Superviosr.astream_async_nolink
and Task.await_many

Reply:
when we reaches a successful threshold , depending upon nb of nodes